### PR TITLE
Feature/components DAO-421 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- TSDX react-storybook template used

### CODE REVIEW

File: packages/ui-components/.storybook/preview.js [hunk 1]
```diff
@@ -1,9 +1,5 @@
...
-  actions: { argTypesRegex: "^on[A-Z].*" },
-  controls: {
-    matchers: {
-      color: /(background|color)$/i,
-      date: /Date$/,
-    },
-  },
-}
\ No newline at end of file
+  actions: {argTypesRegex: '^on.*'},
```
Comment (WissenIstNacht) [inline packages/ui-components/.storybook/preview.js]: Could you elaborate on this change?
Would this not match too many (potentially unwanted) strings?

**Review by WissenIstNacht — APPROVED** [review]

File: packages/ui-components/example/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (WissenIstNacht) [inline packages/ui-components/example/index.html]: Do we need an explicit `example` in this package? Isn't storybook in itself already an example?

File: packages/ui-components/.storybook/preview.js (hunk 1 above)
Comment (Fabricevladimir) [reply packages/ui-components/.storybook/preview.js]: This wasn't an intentional change. It does match too many yes

File: packages/ui-components/example/index.html (hunk 2 above)
Comment (Fabricevladimir) [reply packages/ui-components/example/index.html]: Hmm. This does seem unnecessary.

**Review by sepehr2github — APPROVED** [review]

**Pull request merged**